.stylelintrc.json

```diff
@@ -13,6 +16,8 @@
       "max-nesting-depth": null,
       "selector-max-compound-selectors": null,
 
-      "color-hex-case": "upper"
+      "color-hex-case": "upper",
+
+      "scss/at-import-partial-extension-blacklist":"never"
```

Comment:
Let's include a comment as to why we made this change, for the future devs!

Reply:
I don't think json files support commenting

Reply:
Ah great point, thought we had switched it to the `.js` file already. Let's table that 'till later, then.